pkgs/tools/misc/git-town/default.nix

```diff
@@ -0,0 +1,18 @@
+let 
+  sha_hash = (import ./sha.nix);
+  version = (import ./version.nix);
```

Comment:
Are these indirections really necessary ? Evaluating the name of the package requires opening one more file.

Reply:
Nope! Removed